pkg/virt-launcher/virtwrap/access-credentials/access_credentials_test.go

```diff
@@ -273,9 +273,7 @@ var _ = Describe("AccessCredentials", func() {
 		mockDomain.EXPECT().Free().Times(1)
 
 		Expect(manager.HandleQemuAgentAccessCredentials(vmi)).To(Succeed())
-		DeferCleanup(func() {
-			manager.Stop()
-		})
+		defer manager.Stop()
```

Comment:
Couldn't the second iteration still outpace the defer?

Reply:
I'm not sure I know what you mean. Can you describe it in more detail? When would the second iteration be triggered?